addons/website_slides/static/src/xml/slide_quiz_create.xml

```diff
@@ -5,11 +5,13 @@
         <div t-attf-class="o_wsildes_quiz_question_input mt-3 #{!widget.update ? 'col' : ''}" t-att-data-id="widget.question.id || ''">
             <form class="mb-3">
                 <div class="o_wslides_quiz_question row align-items-center mr-0 mb-2">
-                    <small class="text-muted col-1 text-center">
-                        <span class="h4 o_wslides_quiz_question_sequence"><t t-esc="widget.sequence"/></span>.
-                    </small>
-                    <input type="text" name="question-name" class="form-control col-11" placeholder="Enter your question"
-                           t-att-value="widget.question.text" />
+                    <div class="input-group o_wslides_quiz_form_input_question_display">
```

Comment:
`ml-3` looks like it's doing the trick for both large and mobile views.
Can you try it instead of your `o_wslides_quiz_form_input_question_display` class?

We try to reduce custom css as much as possible.